pollination/honeybee_radiance/multiphase.py

```diff
@@ -0,0 +1,58 @@
+from dataclasses import dataclass
+from pollination_dsl.function import Function, command, Inputs, Outputs
+
+
+@dataclass
+class ViewMatrix(Function):
+    """Calculate view matrix for a receiver file."""
+
+    radiance_parameters = Inputs.str(
+        description='Radiance parameters. -I, -c 1 and -aa 0 are already included in '
+        'the command.', default=''
+    )
+
+    fixed_radiance_parameters = Inputs.str(
+        description='Radiance parameters. -I, -c 1 and -aa 0 are already included in '
+        'the command.', default='-aa 0'
+    )
+
+    sensor_count = Inputs.int(
+        description='Number of maximum sensors in each generated grid.',
+        spec={'type': 'integer', 'minimum': 1}
+    )
+
+    receiver_file = Inputs.file(
+        description='Path to a receiver file.', path='receiver.rad',
+        extensions=['rad']
+    )
+
+    sensor_grid = Inputs.file(
+        description='Path to sensor grid files.', path='grid.pts',
+        extensions=['pts']
+    )
+
+    scene_file = Inputs.file(
+        description='Path to an octree file to describe the scene.', path='scene.oct',
+        extensions=['oct']
+    )
+
+    receivers_folder = Inputs.folder(
+        description='Folder containing any receiver files needed for ray tracing.',
+        path='receivers'
+    )
```

Comment:
I am guessing from this that this PR isn't really ready for review yet? Or maybe I'm just missing where this folder gets used by the command?

Reply:
This folder is similar to the BSDF folder. It needs to be there for the command to work but you don't see it when you check the command itself.